<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HistoryWidget</class>
 <widget class="QWidget" name="HistoryWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>666</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="focusPolicy">
        <enum>Qt::TabFocus</enum>
       </property>
       <property name="text">
        <string>This page displays the history of your successful authentications. Double-click on a provider for more information. You can delete parts or the entire history. You can also save the history as a PDF file.</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="focusPolicy">
          <enum>Qt::TabFocus</enum>
         </property>
         <property name="text">
          <string>Search:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="historySearch">
         <property name="accessibleName">
          <string>Please enter your search</string>
         </property>
         <property name="placeholderText">
          <string>Please enter your search</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableWidget" name="historyTableWidget">
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustToContents</enum>
       </property>
       <property name="tabKeyNavigation">
        <bool>false</bool>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectItems</enum>
       </property>
       <property name="verticalScrollMode">
        <enum>QAbstractItemView::ScrollPerPixel</enum>
       </property>
       <property name="horizontalScrollMode">
        <enum>QAbstractItemView::ScrollPerPixel</enum>
       </property>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>150</number>
       </attribute>
       <attribute name="horizontalHeaderHighlightSections">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="noResultWidget" native="true">
       <layout class="QVBoxLayout" name="noResultsLayout">
        <property name="topMargin">
         <number>5</number>
        </property>
        <item>
         <widget class="QLabel" name="noResultsLabel">
          <property name="text">
           <string>No matching history entries were found. Please modify your search criteria.</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="focusPolicy">
          <enum>Qt::TabFocus</enum>
         </property>
         <property name="accessibleName">
          <string>save history:</string>
         </property>
         <property name="text">
          <string>History:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="saveHistoryCheckBox">
         <property name="accessibleName">
          <string>save history</string>
         </property>
         <property name="accessibleDescription">
          <string/>
         </property>
         <property name="text">
          <string>save</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="historyDeleteButton">
         <property name="text">
          <string>Delete history...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="historyExportButton">
         <property name="accessibleName">
          <string>save history as PDF</string>
         </property>
         <property name="text">
          <string>Save as PDF...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
